goldens/public-api/angular_devkit/schematics/index.md

```diff
@@ -234,6 +235,8 @@ export class DelegateTree implements Tree_2 {
     // (undocumented)
     read(path: string): Buffer | null;
     // (undocumented)
```

Comment:
Question: `readJSON` is documented, so why is it listed as `(undocumented)` here?

Reply:
looks like api extractor doesn't look at the inheritance hierarchy.